react-basic-practice/src/components/SideBar/index.jsx

```diff
@@ -5,16 +5,22 @@ import SelectGroup from '@/components/SelectGroup';
 import './index.css';
 
 const options = [
-  'Name',
-  'Price',
+  { value: 'name ascending', label: 'Name (ascending)' },
```

Comment:
```
{
   label: 'Name (ascending)',
   field: 'name',
   order: 'asc'
}
```

This way you can avoid duplicate code in `handleSort ` method.

Reply:
but when passing value can only pass 1 field (name or order) but while we need both